src/components/scorecard/ScorecardOptInForm.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { SubmitHandler, useForm } from "react-hook-form";
+
+import { useScorecardOptIn } from "./hooks/useScorecardOptIn";
+import { scorecardOptInSchema, ScorecardOptInSchema } from "./models/ScorecardOptInSchema";
+import { getFieldClassName } from "../general/services/getFieldClassName";
+import { useNotifications } from "../../hooks/useNotifications";
+import "./css/scorecard-form.css";
+
+export const ScorecardOptInForm = () => {
+  const { addNotification } = useNotifications();
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm<ScorecardOptInSchema>({
+    resolver: zodResolver(scorecardOptInSchema),
+  });
+
+  const {
+    mutate: postScorecardOptIn,
+    error: backendErrors,
+    isSuccess,
+    isLoading,
+  } = useScorecardOptIn();
+
+  const onSubmit: SubmitHandler<ScorecardOptInSchema> = (formValues) => {
+    postScorecardOptIn(formValues, {
+      onSuccess: () => {
+        addNotification("Thanks for contacting us!", {
+          appearance: "success",
+        });
+      },
+      onError: (err: Error) => {
+        addNotification("There was a problem, please try again later.", {
+          appearance: "error",
+        });
+      },
+    });
+  };
+
+  const blockClassName = "scorecard-form";
+  const errorClassPrefix = `${blockClassName}__input`;
+
+  const successContent = "Thank you! Check your inbox for the scorecard."
+
+  const formContent = (
+    <form
+      className="scorecard-form__form"
+      data-netlify="true"
+      name="scorecardOptIn"
+      onSubmit={handleSubmit(onSubmit)}
+    >
+      <input
+        className={getFieldClassName({
+          errorClassPrefix,
+          errors,
+          fieldName: "firstName",
+          className: "scorecard-form__input",
+        })}
+        placeholder="Your first name"
+        type="text"
+        id="firstName"
+        {...register("firstName", { required: true })}
```

Comment:
with zod you should be able to specify this as required within the zod resolver. You should not have to add that clause here or in the other field.